state/mysql/mysql.go

```diff
@@ -528,28 +663,29 @@ func (m *MySQL) setValue(parentCtx context.Context, querier querier, req *state.
 
 	if req.Options.Concurrency == state.FirstWrite && (req.ETag == nil || *req.ETag == "") {
 		// With first-write-wins and no etag, we can insert the row only if it doesn't exist
-		query := fmt.Sprintf(
-			`INSERT INTO %s (value, id, eTag, isbinary) VALUES (?, ?, ?, ?);`,
-			m.tableName, // m.tableName is sanitized
-		)
-		result, err = querier.ExecContext(ctx, query, enc, req.Key, eTag, isBinary)
+		query = `INSERT INTO %[1]s (value, id, eTag, isbinary, expiredate) VALUES (?, ?, ?, ?, %[2]s)`
```

Comment:
Just realized this could have the same issues with rows that are expired (but not yet GC'd).

If the row is expired, a first-write-wins should work. In this case, it would fail.

Reply:
I found a fix - need this merged soon :)